### PR TITLE
Fe 693 Frequency Plans for our stations 

### PR DESCRIPTION
**Pull request by ipanagiv**

### CODE REVIEW

File: docs/wxm-devices/frequency-plans.mdx [hunk 1]
```diff
@@ -0,0 +1,392 @@
+---
+title: Frequency Plans
+hide_title: true
```
Comment (PavlosTze) [inline docs/wxm-devices/frequency-plans.mdx:3]: Remove this attribute.

**Review by PavlosTze — CHANGES_REQUESTED** [review]
The table looks great, just a minor issue regarding the title and we are good to go.

File: docs/wxm-devices/frequency-plans.mdx (hunk 1 above)
Comment (ipanagiv) [reply docs/wxm-devices/frequency-plans.mdx]: removed

File: docs/wxm-devices/frequency-plans.mdx [hunk 2]
```diff
@@ -0,0 +1,394 @@
...
+| Honduras           | HN           | AU915_SB1     | US915           | AU915            |
+| Hong Kong          | HK           | AS923_1       | US915           | AS923_1          |
+| Hungary            | HU           | EU868         | EU868           | EU868            |
+
+### I
+
+| Country            | Country Code | ISM Frequency |D1/M5/Pulse Freq. | H1/H2 Freq. |
+|--------------------|--------------|---------------|-----------------|------------------|
+| Iceland            | IS           | EU868         | EU868           | EU868            |
+| India              | IN           | IN865         | IN865           | IN865            |
```
Comment (PavlosTze) [inline docs/wxm-devices/frequency-plans.mdx:165]: The second column should be EU868 per the Outline doc.

**Review by PavlosTze — CHANGES_REQUESTED** [review]

File: docs/wxm-devices/frequency-plans.mdx [hunk 3]
```diff
@@ -0,0 +1,394 @@
...
+
+### G
+
+| Country            | Country Code | ISM Frequency |D1/M5/Pulse Freq. | H1/H2 Freq. |
+|--------------------|--------------|---------------|-----------------|------------------|
+| Gabon              | GA           | EU868         | EU868           | EU868            |
+| Gambia             | GM           | EU868         | EU868           | EU868            |
+| Georgia            | GE           | EU868         | EU868           | EU868            |
+| Germany            | DE           | EU868         | EU868           | EU868            |
+| Ghana              | GH           | EU868         | EU868           | EU868            |
```
Comment (PavlosTze) [inline docs/wxm-devices/frequency-plans.mdx:138]: Please update it to the latest Ghana values from the Outline.

**Review by PavlosTze — CHANGES_REQUESTED** [review]

File: docs/wxm-devices/frequency-plans.mdx [hunk 4]
```diff
@@ -0,0 +1,394 @@
...
+| Gibraltar          | GI           | EU868         | EU868           | EU868            |
+| Greece             | GR           | EU868         | EU868           | EU868            |
+| Greenland          | GL           | EU868         | EU868           | EU868            |
+| Grenada            | GD           | AU915_SB1     | US915           | AU915            |
+| Guadeloupe         | GP           | EU868         | EU868           | EU868            |
+| Guam               | GU           | US915         | US915           | US915            |
+| Guatemala          | GT           | AU915_SB1     | US915           | AU915            |
+| Guernsey           | GG           | EU868         | EU868           | EU868            |
+| Guinea             | GN           | EU868         | EU868           | EU868            |
```
Comment (PavlosTze) [inline docs/wxm-devices/frequency-plans.mdx:147]: Wrong, please update it.

File: docs/wxm-devices/frequency-plans.mdx [hunk 5]
```diff
@@ -0,0 +1,394 @@
...
+| Kyrgyzstan         | KG           | EU868         | EU868           | EU868            |
+
+### L
+
+| Country            | Country Code | ISM Frequency |D1/M5/Pulse Freq. | H1/H2 Freq. |
+|--------------------|--------------|---------------|-----------------|------------------|
+| Laos               | LA           | AS923_1       | US915           | AS923_1          |
+| Latvia             | LV           | EU868         | EU868           | EU868            |
+| Lebanon            | LB           | EU868         | EU868           | EU868            |
+| Lesotho            | LS           | EU868         | EU868           | EU868            |
```
Comment (PavlosTze) [inline docs/wxm-devices/frequency-plans.mdx:202]: Wrong, please update it.

File: docs/wxm-devices/frequency-plans.mdx [hunk 6]
```diff
@@ -0,0 +1,394 @@
...
+### M
+
+| Country            | Country Code | ISM Frequency |D1/M5/Pulse Freq. | H1/H2 Freq. |
+|--------------------|--------------|---------------|-----------------|------------------|
+| Macao              | MO           | AS923_1       | US915           | AS923_1          |
+| Madagascar         | MG           | EU868         | EU868           | EU868            |
+| Malawi             | MW           | EU868         | EU868           | EU868            |
+| Malaysia           | MY           | AS923_1       | US915           | AS923_1          |
+| Maldives           | MV           | EU868         | EU868           | EU868            |
+| Mali               | ML           | EU868         | EU868           | EU868            |
```
Comment (PavlosTze) [inline docs/wxm-devices/frequency-plans.mdx:218]: Wrong, please update it.

File: docs/wxm-devices/frequency-plans.mdx [hunk 7]
```diff
@@ -0,0 +1,394 @@
...
+| Malta              | MT           | EU868         | EU868           | EU868            |
+| Marshall Islands   | MH           | AU915_SB1     | US915           | AU915            |
+| Martinique         | MQ           | EU868         | EU868           | EU868            |
+| Mauritania         | MR           | EU868         | EU868           | EU868            |
+| Mauritius          | MU           | EU868         | EU868           | EU868            |
```
Comment (PavlosTze) [inline docs/wxm-devices/frequency-plans.mdx:223]: Wrong, please update it.

File: docs/wxm-devices/frequency-plans.mdx [hunk 8]
```diff
@@ -0,0 +1,394 @@
...
+| Mayotte            | YT           | EU868         | EU868           | EU868            |
+| Mexico             | MX           | AU915_SB2     | US915           | AU915            |
+| Micronesia         | FM           | AU915_SB2     | US915           | AU915            |
+| Moldova            | MD           | EU868         | EU868           | EU868            |
+| Monaco             | MC           | EU868         | EU868           | EU868            |
+| Montenegro         | ME           | EU868         | EU868           | EU868            |
+| Montserrat         | MS           | AU915_SB1     | US915           | AU915            |
+| Morocco            | MA           | EU868         | EU868           | EU868            |
```
Comment (PavlosTze) [inline docs/wxm-devices/frequency-plans.mdx:231]: Wrong, please update it.

File: docs/wxm-devices/frequency-plans.mdx [hunk 9]
```diff
@@ -0,0 +1,394 @@
...
+|--------------------|--------------|---------------|-----------------|------------------|
+| U.S. Minor Outlying Islands | UM | US915         | US915           | US915            |
+| U.S. Virgin Islands | VI         | US915         | US915           | US915            |
+| Uganda             | UG           | AS923_1       | US915           | AS923_1          |
+| Ukraine            | UA           | EU868         | EU868           | EU868            |
+| United Arab Emirates | AE         | EU868         | EU868           | EU868            |
+| United Kingdom     | GB           | EU868         | EU868           | EU868            |
+| United States      | US           | US915         | US915           | US915            |
+| Uruguay            | UY           | AU915_SB1     | US915           | AU915            |
+| Uzbekistan         | UZ           | EU868         | EU868           | EU868            |
```
Comment (PavlosTze) [inline docs/wxm-devices/frequency-plans.mdx:359]: Wrong, please update it.

**Review by PavlosTze — CHANGES_REQUESTED** [review]

File: docs/wxm-devices/frequency-plans.mdx [hunk 10]
```diff
@@ -0,0 +1,400 @@
...
+| French Guiana      | GF           | EU868         | EU868           | EU868            |
+| French Polynesia   | PF           | EU868         | EU868           | EU868            |
+| French Southern Territories | TF | EU868         | EU868           | EU868            |
+
+### G
+
+| Country            | Country Code | ISM Frequency |D1/M5/Pulse Freq. | H1/H2 Freq. |
+|--------------------|--------------|---------------|-----------------|------------------|
+| Gabon              | GA           | -         | -           | -            |
+| Gambia             | GM           | EU868         | -           | Not supported            |
```
Comment (PavlosTze) [inline docs/wxm-devices/frequency-plans.mdx:138]: EU868 should be EU433

File: docs/wxm-devices/frequency-plans.mdx [hunk 11]
```diff
@@ -0,0 +1,400 @@
...
+| Gaza strip          | PS           | -       | -         | -          |
+| Georgia            | GE           | EU868         | EU868           | EU868            |
+| Germany            | DE           | EU868         | EU868           | EU868            |
+| Ghana              | GH           | EU433         | -           | -            |
```
Comment (PavlosTze) [inline docs/wxm-devices/frequency-plans.mdx:142]: Ghana H1/H2 should be "Not supported"

File: docs/wxm-devices/frequency-plans.mdx [hunk 12]
```diff
@@ -0,0 +1,400 @@
...
+| Lithuania          | LT           | EU868         | EU868           | EU868            |
+| Luxembourg         | LU           | EU868         | EU868           | EU868            |
+
+### M
+
+| Country            | Country Code | ISM Frequency |D1/M5/Pulse Freq. | H1/H2 Freq. |
+|--------------------|--------------|---------------|-----------------|------------------|
+| Macau             | MO           | AS923_1       | US915           | AS923_1          |
+| Madagascar         | MG           | EU868         | EU868           | EU868            |
+| Malawi             | MW           | EU868         | EU868           | EU868            |
```
Comment (PavlosTze) [inline docs/wxm-devices/frequency-plans.mdx:219]: All 3 in Malawi are Unknown in Outline.

File: docs/wxm-devices/frequency-plans.mdx [hunk 13]
```diff
@@ -0,0 +1,400 @@
...
+| Malaysia           | MY           | AS923_1       | US915           | AS923_1          |
```
Comment (PavlosTze) [inline docs/wxm-devices/frequency-plans.mdx:220]: Malaysia ISM Frequency (the first column) is AS923_1B

File: docs/wxm-devices/frequency-plans.mdx [hunk 14]
```diff
@@ -0,0 +1,400 @@
...
+| Maldives           | MV           | EU868         | EU868           | EU868            |
```
Comment (PavlosTze) [inline docs/wxm-devices/frequency-plans.mdx:221]: All 3 columns are marked as Unknown in Outline.

File: docs/wxm-devices/frequency-plans.mdx [hunk 15]
```diff
@@ -0,0 +1,400 @@
...
+| Mali               | ML           | EU433       | -           | Not supported            |
+| Malta              | MT           | EU868         | EU868           | EU868            |
+| Marshall Islands   | MH           | -     | -           | -            |
+| Martinique         | MQ           | EU868         | EU868           | EU868            |
+| Mauritania         | MR           | EU868         | EU868           | EU868            |
+| Mauritius          | MU           | EU433         | -           | Not supported            |
+| Mayotte            | YT           | EU868         | EU868           | EU868            |
+| Mexico             | MX           | AU915_SB2     | US915           | AU915            |
```
Comment (PavlosTze) [inline docs/wxm-devices/frequency-plans.mdx:229]: Mexico is wrong, it has the following attributes:
Mexico | US915 | US915 | US915 |

File: docs/wxm-devices/frequency-plans.mdx [hunk 16]
```diff
@@ -0,0 +1,400 @@
...
+| Micronesia         | FM           | AU915_SB2     | US915           | AU915            |
```
Comment (PavlosTze) [inline docs/wxm-devices/frequency-plans.mdx:230]: Wrong, all 3 columns are marked as Unknown in Outline.

File: docs/wxm-devices/frequency-plans.mdx [hunk 17]
```diff
@@ -0,0 +1,400 @@
...
+| Moldova            | MD           | EU868         | EU868           | EU868            |
+| Monaco             | MC           | EU868         | EU868           | EU868            |
+| Montenegro         | ME           | EU868         | EU868           | EU868            |
+| Montserrat         | MS           | AU915_SB1     | US915           | AU915            |
+| Morocco            | MA           | EU433        | -           | Not supported            |
+| Mozambique         | MZ           | EU868         | EU868           | EU868            |
```
Comment (PavlosTze) [inline docs/wxm-devices/frequency-plans.mdx:236]: All 3 columns are marked as Unknown in Outline.

**Review by PavlosTze — CHANGES_REQUESTED** [review]
Except from my comments below, also the following are missing:
1. Mongolia
2. North Macedonia
3. North Korea

**Review by PavlosTze — APPROVED** [review]

**Pull request merged**